mne/io/constants.py

```diff
@@ -1033,6 +1033,7 @@
 FIFF.FIFF_MNE_EPOCHS_SELECTION     = 3800  # the epochs selection
 FIFF.FIFF_MNE_EPOCHS_DROP_LOG      = 3801  # the drop log
 FIFF.FIFF_MNE_EPOCHS_REJECT_FLAT   = 3802  # rejection and flat params
+FIFF.FIFF_MNE_EPOCHS_RAW_SFREQ     = 3803  # original raw sfreq
```

Comment:
No it has not been added. @larsoner suggested it would take some time. Any idea how to proceed? Should I make a PR to the FIFF standard package?

Reply:
It won't actually take too long, I just wanted to make sure that it's really all we need to add. So let's get this PR to mergeable state, then make the fiff-constants PR. It should only take a day or two to get that in I think since this constant is in the range of constants that MNE "owns"

Reply:
> Just marking a change request so we don't forget to update fiff-constants before (adjusting this PR and then) merging this

Kay I have added a PR to upstream FIFF-Constants